myapp/spec/requests/task_spec.rb

```diff
@@ -14,7 +14,7 @@
 
     context "When URL has argument 'sort'" do
       let!(:first_task) do
-        create(:task, title: '1st', description: 'aaa', priority: 'low', status: 'waiting', user_id: user.id,
+        create(:task, title: '1st', description: 'aaa', priority: 'low', status: 'waiting', user_id: user.id, 
```

Comment:
要らないスペースが入ってますー！

```suggestion
        create(:task, title: '1st', description: 'aaa', priority: 'low', status: 'waiting', user_id: user.id,
```

Reply:
個人的にはこんな感じで引数を並べるほうが好みですがおまかせします！

```suggestion
        create(
          :task,
          title: '1st',
          description: 'aaa',
          priority: 'low',
          status: 'waiting',
          user_id: user.id,
```

Reply:
@craftscat 
同意しかなかったので以下のcommitで修正しました！
[e8ee396](https://github.com/Fablic/training/pull/1525/commits/e8ee3963dbaca596c03c4569ba7cb4beb23a66ea)

Reply:
こちらもですね。
[5fd4487](https://github.com/Fablic/training/pull/1525/commits/5fd4487375a9ae1b9c3f3264ecdaba62b6c61115)